src/main/java/blue/lapis/pore/impl/PoreBanList.java

```diff
@@ -24,38 +24,56 @@
  */
 package blue.lapis.pore.impl;
 
-import org.apache.commons.lang3.NotImplementedException;
+import blue.lapis.pore.Pore;
+import blue.lapis.pore.util.PoreText;
```

Comment:
Code style dictates a newline after this import (between `blue.lapis,pore` and third-party imports).